scripts/core/helpers/item.ts

```diff
@@ -1,3 +1,22 @@
 import {IArticle} from 'superdesk-api';
 
 export const getViewImage = (item: IArticle) => item.renditions?.viewImage || item.renditions?.thumbnail;
+
+export const getThumbnailForItem = (item: IArticle) => {
+    if (!item.associations?.featuremedia) {
+        return null;
+    }
+
+    const {featuremedia} = item.associations;
+
+    if (associationIsArticle(featuremedia)) {
+        return getViewImage(featuremedia);
+    }
+
+    return null;
+};
+
+export function associationIsArticle(a: any): a is IArticle {
+    return a.renditions != null;
```

Comment:
It doesn't seem correct since if you have a text article, renditions aren't required and it would produce a false result. I would use `a._created != null`

It would also be better to limit the type of `a` only to those two types `associationIsArticle(a: IArticle | IRelatedArticle)` because if `_created` would also be present in all other entities.

Reply:
![imagen](https://user-images.githubusercontent.com/4324982/82558630-61ef4500-9b6e-11ea-869b-c2984bf1111e.png)

`_created` is not present in the association. Also, the actual purpose of the function is to use it with associations, so checking renditions being null makes more sense. If we want to check if another thing is an article we won't use `associationIsArticle`. What do u think?

Reply:
> `_created` is not present in the association.

@petrjasek can you check if this is a bug?

Pablo, you could then rely on another field like `_etag`?

> Also, the actual purpose of the function is to use it with associations, so checking renditions being null makes more sense.

If we can easily make a generic method `isArticle(a: IArticle | IRelatedArticle): a is IArticle` which does the type checking and works in all cases, not only in associations, I don't see a reason to opt-out for a more limited solution.